recipes/simd/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.microsoft import is_msvc
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class SimdConan(ConanFile):
+    name = "simd"
+    description = "C++ image processing and machine learning library with using of SIMD"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ermig1979/Simd"
+    topics = ("simd", "sse", "avx", "avx-512", "amx", "vmx", "vsx", "neon")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_avx512": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_avx512": False,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+        if self.settings.compiler == "clang":
+            raise ConanInvalidConfiguration(f"{self.ref} don't support clang. (yet)")
+        if is_msvc(self):
+            raise ConanInvalidConfiguration(f"{self.ref} don't support msvc. (yet)")
```

Comment:
@toge The original README reads:

 ```
The library supports dynamic and static linking, 32-bit and 64-bit Windows and Linux, MSVS, G++ and Clang compilers, MSVS project and CMake build systems.
```
Even for Windows:
```
To build the library and test application for Windows 32/64 you need to use Microsoft Visual Studio 2022 (or 2015/2017/2019)
```
So it seems that Clang and MSVC are supported.

Reply:
@franramirez688 
Sorry I missed your comment too long time.
Thank you for your review!

I update this recipe and remove checking environments. (In MSVC, this PR uses MSBuild)

Please review it again.